tests/src/Kernel/LinkFieldProcessorTest.php

```diff
@@ -1,27 +0,0 @@
-<?php
```

Comment:
This test was only needed when we were altering tmgmt to use the default field processor.